Guerrero_Melissa/Assignments/average_list.py

```diff
@@ -0,0 +1,2 @@
+a = [1, 2, 5, 10, 255, 3]
+print sum(a) / 6
```

Comment:
Show me how you would calculate avg without using `sum()` method and instead only using a for-loop